Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:5.10.0
```

Comment:
I usually work on top of official Docker images, but if it's needed i can change the base image on the commit to **mhart/alpine-node**